proseco/fid.py

```diff
@@ -122,7 +126,7 @@ def acqs(self, val):
 
     @property
     def t_ccd(self):
-        # For fids use the guide CCD temperature
+        # For fids use the guide CCD temperature to scale the dark map
```

Comment:
I'm pretty sure that's how t_ccd uses, but the logic in here is a little twisty.